ykman/cli/__main__.py

```diff
@@ -194,11 +194,17 @@ def resolve_device():
 @cli.command('list')
 @click.option('-s', '--serials', is_flag=True, help='Output only serial '
               'numbers, one per line (devices without serial will be omitted).')
+@click.option('-r', '--readers', is_flag=True, help='List available smart card readers.')
 @click.pass_context
-def list_keys(ctx, serials):
+def list_keys(ctx, serials, readers):
     """
     List connected YubiKeys.
     """
+    if readers:
+        for reader in list_readers():
+            click.echo(reader.name)
+        ctx.exit()
```

Comment:
Just a thought: why not print both readers and YubiKeys?

Reply:
Well YubiKeys will show up as well, if the are in CCID mode. I see this as a kind of special case where only reader-names are interesting (NFC or RDP use case), and listing devices again based on descriptors adds noise.

Reply:
Ok :+1: